posts/tests.py

```
@@ -0,0 +1,87 @@
+        even if upvote counts are high.
+        """
+        # Post with 20 upvotes and 0 downvotes
+        positive_post = Post.objects.create(
+            title="Pure Positivity",
+            author=self.author,
+            community=self.community,
+            upvotes=20,
+        )
+
+        # Post with 30 upvotes but 40 downvotes
+        controversial_post = Post.objects.create(
```

Ruff (0.15.0)

[error] Local variable `controversial_post` is assigned to but never used

Remove assignment to unused variable `controversial_post`

(F841)